ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -608,3 +799,8 @@ Below are a number of links to additional documentation about Ansible AVD and An
 - With [Ansible Tower](https://docs.ansible.com/ansible/2.3/tower.html)
```

Comment:
I would just get rid of the With... wording there.

 - Role documentation for eos_designs
 - Role documentation for eos_cli_config_get
 - RedHat Ansible Tower Documentation
 - etc

Also note that the eos_designs link currently points to the eos_cli_config_gen docs.

Reply:
Fixed